src/vm/amd64/JitHelpers_Fast.asm

```diff
@@ -955,5 +955,29 @@ endif ; _DEBUG
 
 NESTED_END TailCallHelperStub, _TEXT
 
-        end
+PAGE_SIZE equ 1000h
```

Comment:
An interesting side-effect here is that the page size (that we probe) is hard-coded to 0x1000, whereas in PAL builds, the page size is currently dynamic. For >4K pages, we might over-probe. But I suppose that is ok -- better perhaps than burning a register to pass in the page size, or creating extra page size specific helpers.

Reply:
I though about several options here:
1) Have several versions of the helpers (e.g. JIT_StackProbe_0x1000 and JIT_StackProbe_0x10000) on platforms that can have different page sizes (e.g. arm64) and do selection when emitting a call to the helper from JIT.
2) Burn a register and pass a page size from the JIT side (which feels weird to me since JIT asks EE for a page size). It's also going to be tough for find a spare register on x86.
3) Use a stack for passing parameters and basically do the same as in 2).
4) Hard-code the page size as I did

I chose 4 and as a contingency plan if there will be a strong requirement for using "true" page size we can add a logic  that will patch the helper during the process startup and adjust the page size.